### DISCUSSION TITLE
Hi I'm new to Github! I have a question

### DISCUSSION THREAD

**Discussion by biyangdo** [community/community#48747] [New to GitHub]
Hi
I'm new. 
Actually had an account for a while but write nothing and been helped for doing some coding sth.
But my account has been flagged. I don't know why.
Can anyone help me?

**Comment by queenofcorgis** [comment]
Hi there! Apologies for the delay. In matters like this the best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them if you hadn't done so already :)
